README.rst

```diff
@@ -258,96 +272,57 @@ want to instrument. Usage for enabling MySQL instrumentation like below:
 
     import mysql.connector
 
-    INTEGRATIONS = ['mysql', 'postgresql']
+    # Trace both mysql-connection and psycopg2
+    integration = ['mysql', 'postgresql']
 
-    config_integration.trace_integrations(INTEGRATIONS)
+    config_integration.trace_integrations(integration)
 
-    tracer = request_tracer.RequestTracer()
-
-    conn = mysql.connector.connect(user='user', password='password')
-    cursor = conn.cursor()
-    query = 'SELECT 2*3'
-    cursor.execute(query)
```

Comment:
Why removing the examples?

Reply:
Because for the most part users already know how to use these libraries. They just need to know how to turn on integration. In the future, we can look into creating more complete samples to show to off integrations.